universe/auto_syncer.go

```diff
@@ -108,7 +108,10 @@ func (f *FederationEnvoy) Start() error {
 			return NewServerAddrFromStr(a)
 		})
 		err := f.AddServer(serverAddrs...)
-		if err != nil {
+		// On restart, we'll get an error for universe servers already
+		// inserted in our DB, since we can't store duplicates.
+		// We can safely ignore that error.
+		if err != ErrDuplicateUniverse {
```

Comment:
Style nit: can also use `errors.Is` here.

Reply:
Another way to handle this would also just be an noop `UPSERT` into the db. I think either works though.